packages/react-widgets/src/widgets/CategoryWidget.js

```diff
@@ -38,21 +37,24 @@ function CategoryWidget(props) {
     onError,
     wrapperProps
   } = props;
+  const dispatch = useDispatch();
+
+  const isSourceReady = useSelector(
```

Comment:
better rename from isSourceReady to `vieportFeaturesForSourceReady`

Reply:
I thought about it, but it's too long. I think it's understandable by the context that when a source is ready (in the context of widget's data fetching) is when viewport features are ready.

Reply:
fair enough!